pkg/server/proto/server.proto

```diff
@@ -1179,6 +1186,76 @@ message Task {
   // This task info message represented as an Any format. This is the full
   // task encoded directly by the server that has access to the proto to decode it.
   opaqueany.Any state = 6;
+
+  // The overall state of the Task triple. See the enum def for details for
+  // each possible Task state. This will be computed for each GetTaskResponse
+  // based on the Task job ids receieved from the database.
+  State job_state = 7;
+
+  // The resource that gets created to run the task job for this Task
+  // TODO(briancain): This field has not been implemented yet. See
+  // "internal/runner/operation.go" for more info.
+  string resource_name = 8;
```

Comment:
What is this? I didn't find anything in operation.go.

Reply:
Fixed the comment in 73a9c4c437b3e574722e41c45e4fac357c6f90f5